coop_account/models/account_journal.py

```diff
@@ -0,0 +1,8 @@
+
+from odoo import models, fields
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    export_wrong_reconciliation = fields.Boolean()
```

Comment:
Can you please add a `help=` attribute with some kind of explaination to what it's doing?

Reply:
@ivantodorovich : done